src/objset.c

```diff
@@ -721,8 +727,8 @@ static void ResizeObjMap(Obj map, UInt bits)
   for (i = 0; i < size; i++) {
     Obj obj = READ_SLOT(map, i*2);
     if (obj && obj != Undefined) {
-      AddObjMapNew(new, obj,
-        READ_SLOT(map, i*2+1));
+        AddObjMapNew(new, obj, READ_SLOT(map, i * 2 + 1));
+        CHANGED_BAG(new);
```

Comment:
This is not needed as no GC can occur in this loop, and afterwards SwapMasterPointer ensures both bags are marked aa changed (so the two CHANGED_BAG calls after it could also go)